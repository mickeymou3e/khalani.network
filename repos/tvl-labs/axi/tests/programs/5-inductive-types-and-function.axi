../../tutorial/5-inductive-types-and-function.axi