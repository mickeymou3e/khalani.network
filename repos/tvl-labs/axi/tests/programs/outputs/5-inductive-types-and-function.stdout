Result: Nothing
Result: Nothing
Result: Proved(forall Nat. =[add[@0:Nat, Zero:Nat]):Nat, @0:Nat])
Result: Proved(forall Nat. forall Nat. =[add[@1:Nat, Succ[@0:Nat]):Nat]):Nat, Succ[add[@1:Nat, @0:Nat]):Nat]):Nat])
Result: Nothing
Result: Proved(forall Nat. =[add[Zero:Nat, @0:Nat]):Nat, @0:Nat])
